site/it/tutorials/keras/overfit_and_underfit.ipynb

```diff
@@ -913,7 +913,7 @@
         "id": "Fy3CMUZpzH3d"
       },
       "source": [
-        "### Traccia dele perdite in addestramento e validazione"
+        "### Traccia delle funzioni obiettivo in addestramento e validazione"
```

Comment:
or `Tracciato`?

Reply:
"Andamento" is probably better. Thank you